editors/code/package.json

```diff
@@ -418,6 +418,16 @@
                     "default": {
                         "/rustc/<id>": "${env:USERPROFILE}/.rustup/toolchains/<toolchain-id>/lib/rustlib/src/rust"
                     }
+                },
+                "rust-analyzer.debug.openUpDebugPane": {
```

Comment:
```suggestion
                "rust-analyzer.debug.openDebugPane": {
```

Reply:
Was not sure about the proper name. The best one I've come up with - `alwaysShowDebugPane` is too long. Or not?